Formula/qgis3.rb

```diff
@@ -374,41 +409,55 @@ def install
       -DWITH_INTERNAL_QWTPOLAR=FALSE
       -DQGIS_MACAPP_BUNDLE=0
       -DQGIS_MACAPP_INSTALL_DEV=FALSE
-      -DWITH_QSCIAPI=TRUE
+      -DWITH_QSCIAPI=FALSE
       -DWITH_STAGED_PLUGINS=TRUE
       -DWITH_CUSTOM_WIDGETS=TRUE
       -DWITH_ASTYLE=FALSE
-      -DUSE_CCACHE=OFF
+      -DWITH_GRASS=FALSE
```

Comment:
Is this the solution to the Grass windowing problem?

Reply:
No, that disables GRASS 6, the spoorte we use is WITH_GRASS7.
The problem was in some `def` and `path`.

Reply:
You can see that it is used in the same way as in `qgis2`.

https://github.com/OSGeo/homebrew-osgeo4mac/blob/17bf22a08f46f7a17bac9b7f5615c26157283d04/Formula/qgis2.rb#L285

Reply:
Ok, that makes sense. My apologies.

So in the PR, what is the actual fix to the windowing issue?